packages/glibc.rb

```diff
@@ -41,19 +44,22 @@ class Glibc < Package
        x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997'
     })
   when '2.32' # All architectures with updates past M92.
+    version '2.32-1'
+    compatibility 'x86_64 aarch64 armv7l'
```

Comment:
These lines don't need to be here.  Since release 92 is not stable yet, I think we can get away with not incrementing the version.

Reply:
I'm not sure we should set a precedent of not incrementing a deployed version?

It's one thing to not officially support non-release channel, it's another to intentionally break non-release channel by not showing appropriate updates.

(Also, with the M91 breakage this week, it has been often suggested that people move to beta channel, so we might have users with M92 who need to get the newer build of Glibc 2.32.)

Reply:
It only sets a bad precedent if it's on the stable branch.  Since it isn't, I think we can get away with not incrementing.  I can merge this but you could clean this up on the next update by going back to just 2.32 for the version and set it at the top.